neptune/new/run.py

```diff
@@ -406,6 +412,10 @@ def sync(self, wait: bool = True) -> None:
             for attribute in attributes:
                 self._define_attribute(parse_path(attribute.path), attribute.type)
 
+    @property
+    def root(self):
+        return Handler(self, "")
```

Comment:
@Herudaio what do you think? Should it be public?
I don't think we can drop assign method in `Run` because of backward compatibility. And not having fetch method here is inconsistent.

Reply:
added public `Run.fetch` in https://github.com/neptune-ai/neptune-client/pull/549/commits/c49abb3cfddc3a289bbeb2b566eb15e34d4681a7 , but I'd like to keep this `root` thing -- I believe it's a nice abstraction for `Handler(self, "")`

Reply:
Don't have a strong opinion here. 
Food for thought: It's a public API so anything we add here for our own sake we cannot add for sake of users. If you can think of 1+ use case for property or method called root that could be useful for en user then let's not add it as public property.

Reply:
I though it would be nicer not to handle each method (like `fetch()`) both in Handler and Run.

Now that I'm writing it, `run.fetch() , run["ns"].fetch()` is much nicer than `run.root.fetch() , run["ns"].fetch()` (even if implementation-wise it's dirtier IMO). I'll remove `root`.